integration/container/nat_test.go

```diff
@@ -56,6 +56,7 @@ func TestNetworkLocalhostTCPNat(t *testing.T) {
 }
 
 func TestNetworkLoopbackNat(t *testing.T) {
+	skip.If(t, testEnv.GitHubActions, "FIXME: broken on GitHub Runner")
```

Comment:
We probably should create tracking issues for these

Reply:
I put related issues in PR description, not enough?

Reply:
Could you also add the (link to) description to the comment lines